### ISSUE TITLE
 CSS Grid vs Flex video

### ISSUE THREAD

**Issue by manojna191** [girlscript/winter-of-contributing#3496]
### Description

Video with presentation

- Grid:

1. Grid definition
2. How grid looks 

- Flex:

1. Flex definition
2. how flex looks 

- Difference between grid and flex

- Advantages and Disadvantages of flex and grid

- When to use grid and flex

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Video

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by Divyanshigarg** [comment]
@manojna191  are u making video about the topic ?

**Comment by manojna191** [comment]
Yes I am why??  Sent from Mail for Windows From: Divyanshi Garg 27Sent: 01 October 2021 15:45To: girlscript/winter-of-contributingCc: manojna191; MentionSubject: Re: [girlscript/winter-of-contributing] CSS Grid vs Flex video ***@***.*** are u making video about the topic ?—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by manojna191** [comment]
yes I am

**Comment by manojna191** [comment]
@Divyanshigarg yes I am
